templates/prometheus.go

```diff
@@ -40,36 +40,6 @@ var (
 	PrometheusServingCertSecretName     string = "prometheus-serving-cert-secret"
 	PrometheusKubeRBACPoxyConfigMapName string = "prometheus-kube-rbac-proxy-config"
 )
-var metrics = []string{
-	"job:ceph_versions_running:count",
-	"job:ceph_pools_iops_bytes:total",
-	"job:ceph_pools_iops:total",
-	"job:kube_pv:count",
-	"job:ceph_osd_metadata:count",
-	"ceph_health_status",
-	"ceph_cluster_total_used_raw_bytes",
-	"ceph_cluster_total_bytes",
-	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
-	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
-}
-
-var alerts = []string{
-	"CephMdsMissingReplicas",
-	"CephMgrIsAbsent",
-	"CephMgrIsMissingReplicas",
-	"CephNodeDown",
-	"CephClusterErrorState",
-	"CephClusterWarningState",
-	"CephOSDVersionMismatch",
-	"CephMonVersionMismatch",
-	"CephOSDFlapping",
-	"CephOSDDiskNotResponding",
-	"CephOSDDiskUnavailable",
-	"CephDataRecoveryTakingTooLong",
-	"CephPGRepairTakingTooLong",
-	"CephMonQuorumAtRisk",
-	"CephMonQuorumLost",
-}
```

Comment:
Please put this section in a comment instead of removing it. We will need to reintroduce it very soon.

Reply:
Added it back in the commit where is was removed

Reply:
@leelavg This is a change that overrides entire files and messes up the history. I have asked specifically to have this code in to make it easier on the revert in the next PR